code/game/turfs/unsimulated/floor.dm

```diff
@@ -120,12 +122,13 @@
 		return BUILD_SUCCESS
 	return BUILD_FAILURE
 
-/turf/unsimulated/floor/snow/ChangeTurf()
-	..()
-	for(var/direction in alldirs)
-		var/turf/adj_tile = get_step(src, direction)
-		if(istype(adj_tile,/turf/unsimulated/floor/snow))
-			adj_tile.update_icon()
+/turf/unsimulated/floor/snow/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_lighting_update = 0, var/allow = 0)
+	..(N, tell_universe, force_lighting_update, 1) // yeah you're allowed FOR NOW.
+	if(!allow) // IF YOU'RE ALLOWED YOU AREN'T ACTUALLY ALLOWED IT'S ALL A RUSE AND I LIED (only shuttles use allow = 1 and we don't want it to update with shuttles)

```

Comment:
This specific part of the code needs to be commented, as having something happen if !allow doesn't make too much sense, honestly.